docs/Supported-OS-Types.md

```diff
@@ -146,6 +146,7 @@
 |QTECH               |QSW-2800, QSW-3400, QSW-3450, QSW-3500|[qtech](/lib/oxidized/model/qtech.rb)
 |Quanta              |Quanta / VxWorks 6.6 (1.1.0.8)|[quantaos](/lib/oxidized/model/quantaos.rb)
 |Siklu               |EtherHaul                     |[siklu](/lib/oxidized/model/siklu.rb)
+|                    |Multihaul TG                  |[siklumhtg](/lib/oxidized/model/siklumhtg.rb)
```

Comment:
Please add a link to your model note.

Maybe you could consider becoming a model maintainer?

Reply:
I reviewed the model maintainer role when I was working on this PR. I'm not much of a developer, but I'll try. At a minimum, I can test any changes someone might submit for the models. I listed myself as the maintainer for both models and linked to the note.